src/renderer/src/routes/pages/Wallets/AccountOverview/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { useEffect, useMemo, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { TbStepInto, TbStepOut } from 'react-icons/tb'
+import { useNavigate, useParams } from 'react-router-dom'
+import { IAccountState } from '@renderer/@types/store'
+import { BalanceChart } from '@renderer/components/BalanceChartPanel/BalanceChart'
+import { Button } from '@renderer/components/Button'
+import { Separator } from '@renderer/components/Separator'
+import { BalanceHelper } from '@renderer/helpers/BalanceHelper'
+import { FilterHelper } from '@renderer/helpers/FilterHelper'
+import { useAccountsSelector } from '@renderer/hooks/useAccountSelector'
+import { useBalancesAndExchange } from '@renderer/hooks/useBalancesAndExchange'
+
+export const AccountOverview = () => {
+  const { t } = useTranslation('pages', { keyPrefix: 'accountOverview' })
+  const { t: commonGeneral } = useTranslation('common', { keyPrefix: 'general' })
+  const navigate = useNavigate()
+  const { address } = useParams()
+  const { accounts } = useAccountsSelector()
+  const [selectedAccount, setSelectedAccount] = useState<IAccountState>()
```

Comment:
You can use a useMemo here instead.
```suggestion
    const account = useMemo(() => accounts.find(account => account.address === address), [accounts, address])

```